PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.h

```diff
@@ -18,7 +18,7 @@
 class AliFemtoDreamTrackHist {
  public:
   AliFemtoDreamTrackHist();
-  AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool TOFM, float pTmin = 0.5, float pTmax = 4.05,  int MultRangeLow = 27, int MultRangeHigh = 55);
+  AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool TOFM,bool TOFMSq, float pTmin = 0.5, float pTmax = 4.05,  int MultRangeLow = 27, int MultRangeHigh = 55);
```

Comment:
Here please put the bool TOFMSq as last argument and put it to false, for backward compatibility